modules/developers-guide/pages/work-with-languages.adoc

```diff
@@ -1,23 +1,23 @@
 = Develop using different languages
 :cpp: C++
 
-Most of the examples in this guide use {proglang}—the programming language specifically designed to work with the {IC}. 
-Potentially, however, you can write programs in any language that compiles to WebAssembly to deploy applications that run on the {IC}.
-This section provides some high-level guidance for writing programs in different languages and how to deploy them on the {IC}.
+Most of the example dapps in this guide use {proglang}—the programming language specifically designed to work with the {IC}. 
+Potentially, however, you can write dapps in any language that compiles to WebAssembly to deploy applications that run on the {IC}.
+This section provides some high-level guidance for writing dapps in different languages and how to deploy them on the {IC}.
 
 == Using Rust
 
-You can create Rust projects to run on the {IC} by using Cargo and compiling your program to use WebAssembly as the target output.
+You can create Rust projects to run on the {IC} by using Cargo and compiling your dapp to use WebAssembly as the target output.
 
-This section provides a summary of the key steps involved in deploying a Rust program as a smart contract canister on the {IC}.
+This section provides a summary of the key steps involved in deploying a Rust program as a canister smart contract on the {IC}.
 You should note, however, that the steps described here only illustrate one approach. 
 Other implementation approaches are also possible. 
 
-Note that the link:https://github.com/dfinity/cdk-rs[Rust canister development kit (Rust CDK)] for provides some shortcuts to make it easier to write functions as query and update calls and includes several link:https://github.com/dfinity/cdk-rs/tree/next/examples[examples] to get you started building Rust-based projects, but you can also develop applications for the {IC} without using the Rust CDK.
+Note that the link:https://github.com/dfinity/cdk-rs[Rust canister development kit (Rust CDK)] for provides some shortcuts to make it easier to write functions as query and update calls and includes several link:https://github.com/dfinity/cdk-rs/tree/next/examples[examples] to get you started building Rust-based projects, but you can also develop dapps for the {IC} without using the Rust CDK.
 
 === Create a project
 
-Because most Rust programmers use Cargo to handle build and package management tasks—such as downloading and compiling the libraries your program depends on—your first step is to create a new Rust project using the Cargo command-line interface.
+Because most Rust programmers use Cargo to handle build and package management tasks—such as downloading and compiling the libraries your dapp depends on, your first step is to create a new Rust project using the Cargo command-line interface.
```

Comment:
```suggestion
Because most Rust programmers use Cargo to handle build and package management tasks, such as downloading and compiling the libraries your dapp depends on, your first step is to create a new Rust project using the Cargo command-line interface.
```